tests/lib/nm/ipv6_test.py

```diff
@@ -29,6 +29,7 @@ def NM_mock():
 
 
 def test_create_setting_without_config(NM_mock):
+    NM_mock.SettingIP6Config.new().props.addresses = []
```

Comment:
nit: Please add a newline. (separate setup, the tested action and the assert using newlines, so it will be a bit more clear which is which)